apps/SageAccountWeb/src/components/RegisterAccount1.tsx

```diff
@@ -175,128 +160,125 @@ export const RegisterAccount1 = (props: RegisterAccount1Props) => {
           {page !== Pages.EMAIL_REGISTRATION_THANK_YOU && (
             <>
               <Box sx={{ py: 10, px: 8, height: '100%', position: 'relative' }}>
-                <BackButton />
+                <BackButtonForPage />
                 <Box sx={{ minHeight: '530px' }}>
-                  <div className="mainContent">
-                    <div className="panel-logo logo-wrapper">
-                      <SourceAppLogo />
+                  <div className="panel-logo logo-wrapper">
+                    <SourceAppLogo />
+                  </div>
```

Comment:
The diff for the rest of this file is confusing -- the only difference is removing the wrapping tag `<div className="minContent">`, since the styling applied with this class in _Register.scss is out of date